src/components/Comments/WriteComment.js

```diff
@@ -35,7 +35,7 @@ const WriteComment = ({
   const [profile] = React.useContext(Context).profile;
   const commentInputRef = React.useRef(null);
   const [isAnonimity, setIsAnonimity] = React.useState(false);
-  const isCommentEnabled = value.length > 0;
+  const isCommentEnabled = value?.length > 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is good for preventing potential errors when `value` is null or undefined. The optional chaining operator (`?.`) is used correctly to check if `value` exists before accessing its `length` property.
```diff
- const isCommentEnabled = value.length > 0;
+ const isCommentEnabled = value?.length > 0;
```
This change improves the robustness of the code by handling edge cases where `value` might be `null` or `undefined`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->